python/py_encoder_description.cpp

```diff
@@ -16,13 +20,140 @@ namespace py = pybind11;
 
 using bufr::encoders::Description;
 
+
+template<typename T>
+class PyGlobalWriter : public bufr::encoders::GlobalWriter<T>
```

Comment:
@rmclaren Here, are you adding a Python interface to write global attributes?

Reply:
@emilyhcliu No need too specify zarr, as the description between the different outputs is the same.